freiburg/index.html

```diff
@@ -19,3 +19,5 @@
 ---
 <h3>Aktiv</h3>
 <p>Du bist aus {{ page.title }} oder Umgebung und hast Lust mitzumachen? Super! Dann trag Dich hier ein.</p>
+<p>Das nächste Treffen des OK Lab Freiburg findet am 20. Mai um 19 Uhr im Grünhof (Belfortstr. 52) statt.</p>
```

Comment:
https://github.com/okfde/codefor.de/tree/gh-pages/_termine <- dort einfach die Termine eintragen und dann tauchen die auch in der Liste auf ;)